specification/subscription/resource-manager/Microsoft.Subscription/stable/2020-09-01/subscriptions.json

```diff
@@ -460,6 +460,10 @@
         "subscriptionId": {
           "description": "This parameter can be used to create alias for existing subscription Id",
           "type": "string"
+        },
+        "resellerId": {
```

Comment:
>resellerId [](start = 9, length = 10)

This appears to be a write-only property. Why not return it back out in the response and on GETs?

Reply:
In the PAST we have extensive discussion on this thing, we have other properties in the request body that we are not returning. The reason is those properties are related to subscription not alias and they can change so we decided to not return them